### PR TITLE
Bug fix search

### PR DESCRIPTION
**Pull request by TamariIv**

### CODE REVIEW

File: insertion.py [hunk 1]
```diff
@@ -4,6 +4,7 @@
 import os
 from dataclasses import dataclass
 
+
 @dataclass
```
Comment (yiskalevi) [inline insertion.py:8]: Maybe you should separate the class into a separate file

File: cli.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import argparse
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("data_path", type=str)
+    args = parser.parse_args()
+    data_path = args.data_path
+    init_system(data_path)
+
```
Comment (hodaya12) [inline cli.py]: unnecessary space

File: search_wards.py [hunk 3]
```diff
@@ -63,28 +81,37 @@ def correction_to_input(words: list) -> dict:
...
-        elif x.completed_sentence<y.completed_sentence:
-            return -1
-        return 0
+            list_suggest = search_suggestion(change_words)
+            if list_suggest:
+                complete_sentences[" ".join(change_words)] = list_suggest
+    return complete_sentences
+
+
+# def compare_func(x, y):
```
Comment (yiskalevi) [inline search_wards.py]: delete this

File: search_wards.py [hunk 4]
```diff
@@ -146,27 +184,12 @@ def search_suggestion(words: List) -> List[int]:
         intersection_dict = new_intersection_dict
     return list(intersection_dict.keys())
 
-def main(path="Archive1"):
+
+def init_system(path):
```
Comment (yiskalevi) [inline search_wards.py:186]: Add documentation

File: search_wards.py [hunk 5]
```diff
@@ -15,31 +15,49 @@ def search_word_in_tree(word: str) -> dict:
...
+    """
+    try:
+        return dict_del[sid]
     except:
         return None
 
-def find_in_dict(sid:int)-> insertion.SentenceInfo:
-    return dict_del[sid]
 
 def get_word_corrections(word: str) -> list:
```
Comment (yiskalevi) [inline search_wards.py:38]: Add documentation

File: search_wards.py [hunk 6]
```diff
@@ -146,27 +184,12 @@ def search_suggestion(words: List) -> List[int]:
...
+
+def init_system(path):
     global trie_tree
     global dict_del
     path_list = []
     for subdir, dirs, files in os.walk(path):
         for file in files:
             path_list.append(os.path.join(subdir, file))
     dict_del, trie_tree = insertion.insert(path_list)
-    print("dict_del")
```
Comment (hodaya12) [inline search_wards.py:157]: Do not write print

File: insertion.py (hunk 1 above)
Comment (TamariIv) [reply insertion.py:8]: just a dataclass
i think it might be ok

File: search_wards.py [hunk 7]
```diff
@@ -63,28 +81,37 @@ def correction_to_input(words: list) -> dict:
...
+
+def remove_duplicates(input_list):
+    unique_list = []
+    for item in input_list:
+        if item not in unique_list:
+            unique_list.append(item)
+    return unique_list
 
 
 def get_best_k_completions(prefix: str) -> List[AutoCompleteData]:
```
Comment (TamariIv) [inline search_wards.py:114]: add documentation

File: search_wards.py (hunk 6 above)
Comment (TamariIv) [reply search_wards.py:157]: done

File: search_wards.py (hunk 5 above)
Comment (TamariIv) [reply search_wards.py:38]: added

File: search_wards.py (hunk 4 above)
Comment (TamariIv) [reply search_wards.py:186]: added

File: search_wards.py (hunk 3 above)
Comment (TamariIv) [reply search_wards.py]: deleted

**Pull request merged**